unidist/core/backends/mpi/core/serialization.py

```diff
@@ -53,9 +63,14 @@ def is_pickle5_serializable(data):
     bool
         ``True`` if the data should be serialized with pickle using protocol 5 (out-of-band data).
     """
-    return hasattr(data, "__reduce_ex__") and inspect.ismethod(
-        getattr(data, "__reduce_ex__")
-    )
+    is_pandas_instance = False
+    is_numpy_instance = False
```

Comment:
```suggestion
    is_serializable= False
```

One variable is enough.